### PR TITLE
Fixing broken link in doc + Suggestion for development environment

### PR DESCRIPTION
**Pull request by dineshtrivedi**
Hey @carsongee , this is just a small fix for the link on the Developer section.

I have seen that my previous PR failed with python3.5 so I fixed my dev environment, it was the only python version that I didn't run tox with.

I was wondering if a suggestion for the development environment would fit the documentation.
Let me know what you think, please :)

### CODE REVIEW

**Comment by dineshtrivedi** [comment]
closes #126

File: pytest_pylint/plugin.py [hunk 1]
```diff
@@ -9,10 +9,10 @@
 from configparser import ConfigParser, NoSectionError, NoOptionError
 from os.path import exists, join, dirname
 
-from pylint import lint
-from pylint.config import PYLINTRC
 import pytest
 import toml
+from pylint import lint
```
Comment (carsongee) [inline pytest_pylint/plugin.py:14]: Can you leave these where they are? They're alphabetized. Also, it seems unrelated to the other changes

File: .gitignore [hunk 2]
```diff
@@ -62,7 +62,7 @@ target/
 
 # Virtualenv
 .venv
-.pytest-pylint
+.pytest_pylint
```
Comment (carsongee) [inline .gitignore:65]: I don't think this repo adds this folder. If it's something local to you, could you add it to your global ~/.gitignore instead?

File: pytest_pylint/plugin.py (hunk 1 above)
Comment (dineshtrivedi) [reply pytest_pylint/plugin.py]: Sure. Done

File: .gitignore (hunk 2 above)
Comment (dineshtrivedi) [reply .gitignore]: Sure. Done

File: pytest_pylint/plugin.py (hunk 1 above)
Comment (dineshtrivedi) [reply pytest_pylint/plugin.py]: Sorry, this was my fingers automatic doing ctrl+alt+o in pycharm haha

File: .gitignore (hunk 2 above)
Comment (dineshtrivedi) [reply .gitignore]: I added because I used this folder as a suggestion for virtualenv name in the docs
